### DISCUSSION TITLE
Where did downfall particles dissapeared?

### DISCUSSION THREAD

**Discussion by odvigajlo** [CosmicHikari/Weather2-Remastered#75] [Questions] [answered]
So, in original mod there was downfall, and splash particles for rain, they are pretty nice visual feature. But in this mod i seem to not see them just normal rain, even config misses them. What happened to those?

**Comment by CosmicHikari** [comment]
In your ezconfig gui set all graphics as high as they go and there's loads of new particles you will see.

Hope this helps!

**Reply by odvigajlo** [reply]
no, i still can't see them

**Comment by Mrbt0907** [comment, marked as answer]
> So, in original mod there was downfall, and splash particles for rain, they are pretty nice visual feature. But in this mod i seem to not see them just normal rain, even config misses them. What happened to those?

What you are looking for is under **Advanced Config -> Weather2 Particle -> enable_heavy_precipitation and enable_precipitation_splash**.

**Reply by odvigajlo** [reply]
huh, looks like you changed name of those, because i remember in original one there was downfall instead of heavy preciptiation.

**Comment by CosmicHikari** [comment]
Didn't know this was a thing. Awesome! Thanks for the information.
